CMakeLists.txt

```diff
@@ -1,10 +1,5 @@
 # Projects Settings   
-cmake_minimum_required (VERSION 3.10...3.21)
-
-# allow enabling lto from the command line
-if(POLICY CMP0069)
-    cmake_policy(SET CMP0069 NEW)
-endif()
```

Comment:
I take it this is the default for the min version we require now?

Reply:
Yes, it is for 3.9+